plotextractor/output_utils.py

```diff
@@ -191,6 +191,7 @@ def prepare_image_data(extracted_image_data, output_directory,
             output_directory,
             image_mapping.keys()
         )
+        image_location = os.path.abspath(image_location)
 
         if not image_location or not os.path.exists(image_location) or \
                 len(image_location) < 3:
```

Comment:
I think this is excluding `.` and `..`, but probably doesn't work with your change

Reply:
what do you mean?

Reply:
my guess is that `len(image_location) <3` is probably there to exclude `.` and `..`. Anyway, whatever the reason for it was, by making the path absolute you are changing its length, thus changing the behaviour of this check.

Reply:
.... dammit xd

Reply:
I think the image location always has `.png` and the length check `< 3` is just basic sanity

because `get_image_location()` returns things from
`converted_image_should_be = get_converted_image_name(image)`
which replaces image extensions with `png`

Reply:
That's what I though too, but does not hurt anyhow (also, `.g` will we mapped to the full path to it from the `/` like `/home/dcaro/Work/repos/per_user/inspirehep/inspire-next/.g`, missing that check).

Reply:
so replace with `.endswith('png')`  ?

Reply:
problem is that I'm not sure if it was for that... probably all this code needs some refactor too to make it easier for the next person to understand xd